HidUtil/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.240405.15" targetFramework="native" />
```

Comment:
You're free to propose migration to `CppWinRT` in this project, but that has to be done in a _separate_ PR that is not bundled with the introduction of a self-test PR method.